### PR TITLE
[FIX] No rotate option, to prevent image quality loss

### LINKED ISSUE #11480: Image rotation leads to quality loss
### Description:

Big images are changed/scaled/manipulated during upload

### Steps to reproduce:

1. Drag image file (6000x4500 pixels; 300dpi; RGB; 4.2 MB) in chat (OSX or Firefox browser)
2. Download file from chat

### Expected behavior:

File is not changed/scaled/manipulated (I know it does not save bandwidth, but it breaks our processes)

### Actual behavior:

Downloaded image is not the same (6000x4500 pixels; 72dpi; RGB; 0.5 MB). In the server file system the file size is 0.5MB

### Server Setup Information:

- Version of Rocket.Chat Server: 0.66.2
- Operating System: Ubuntu
- Deployment Method: snap
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: v8.11.3
- MongoDB Version: v3.0.15

### Additional context

Is there an configuration option for this? Account preferences -> save mobile bandwidth = false does not change anything

### Relevant logs:

There are no messages on the browser console.

### PR DESCRIPTION
**Pull request by stleitner**
Closes #11480

### CODE REVIEW

File: app/message-attachments/client/stylesheets/messageAttachments.css [hunk 1]
```diff
@@ -83,6 +83,8 @@ html.rtl .attachment {
 		margin-top: 4px;
 
 		line-height: 0;
+
+		image-orientation: from-image;
```
Comment (sampaiodiego) [inline app/message-attachments/client/stylesheets/messageAttachments.css]: @RocketChat/frontend is this something we can rely on?

Comment (ggazzo) [reply app/message-attachments/client/stylesheets/messageAttachments.css]: no

[the remark this answers is not in the document]
Comment (stleitner) [reply app/message-attachments/client/stylesheets/messageAttachments.css]: If the image-orientation property is not set, the Firefox does not respect the EXIF information of the image

[the remark this answers is not in the document]
Comment (stleitner) [reply app/message-attachments/client/stylesheets/messageAttachments.css]: Chrome will implement EXIF by default in version 79 [https://bugs.chromium.org/p/chromium/issues/detail?id=158753](url)

File: app/file-upload/server/startup/settings.js [hunk 2]
```diff
@@ -24,6 +24,12 @@ settings.addGroup('FileUpload', function() {
 		i18nDescription: 'FileUpload_ProtectFilesDescription',
 	});
 
+	this.add('FileUpload_RotateImages', true, {
```
Comment (rodrigok) [inline app/file-upload/server/startup/settings.js:27]: Missing translation for the key

File: app/file-upload/server/startup/settings.js [hunk 3]
```diff
@@ -24,6 +24,12 @@ settings.addGroup('FileUpload', function() {
...
+		type: 'boolean',
+		public: true,
```
Comment (rodrigok) [inline app/file-upload/server/startup/settings.js:29]: This setting doesn't need to be public since it's used on the server-side only

**Pull request merged**